lib/cli-engine.js

```diff
@@ -298,23 +301,31 @@ function getCacheFile(cacheFile, cwd) {
     }
 
 
-    // in case the file exists we need to verify if the provided path
-    // is a directory or a file. If it is a directory we want to create a file
-    // inside that directory
+    /*
+     * in case the file exists we need to verify if the provided path
+     * is a directory or a file. If it is a directory we want to create a file
+     * inside that directory
+     */
     if (fileStats) {
-        // is a directory or is a file, but the original file the user provided
-        // looks like a directory but `path.resolve` removed the `last path.sep`
-        // so we need to still treat this like a directory
+
+        /*
+         * is a directory or is a file, but the original file the user provided
+         * looks like a directory but `path.resolve` removed the `last path.sep`
+         * so we need to still treat this like a directory
+         */
```

Comment:
Hmmm, the empty line between open parentheses and comments looks strange to me....
And this is just my feeling, I like line-comments than block-comments, since only `/*` or `*/` lines.